atoma-utils/src/hashing.rs

```diff
@@ -3,21 +3,14 @@ use blake2::{
     Blake2b, Digest,
 };
 
-/// Computes a Blake2b hash of the input data
+/// Computes the `BLAKE2b` hash of the provided data
 ///
 /// # Arguments
-/// * `slice` - A byte slice containing the data to be hashed
+/// * `slice` - Data to hash
```

Comment:
I think the previous description was more enlightening

Reply:
Upon re-reading it, I agree with you. 9db8e55